test/semantics/resolve69.f90

```diff
@@ -0,0 +1,45 @@
+subroutine s1()
+  ! C701 (R701) The type-param-value for a kind type parameter shall be a
+  ! constant expression.  This constraint looks like a mistake in the standard.
```

Comment:
Why do you think that C701 is a mistake?  Kind type parameter values must be constant.

Reply:
Good question.  I can't remember why, but when I first started looking at type parameters, I thought I saw an inconsistency and added this comment.  I'll remove it.